pyodide-test-runner/pyodide_test_runner/node_test_driver.js

```diff
@@ -25,8 +24,6 @@ const context = {
   TextDecoder: util.TextDecoder,
   TextEncoder: util.TextEncoder,
   URL,
-  atob: base64.decode,
-  btoa: base64.encode,
```

Comment:
Can you comment why removing it?

Reply:
Those functions were previously required in `run_in_pyodide` decorator, but was removed (#2510).

https://github.com/pyodide/pyodide/blob/7a9273dd6c893ab1273f8c5d22f62f60296f13d5/pyodide-build/pyodide_build/testing.py#L88